typings/index.d.ts

```diff
@@ -1560,6 +1572,7 @@ declare module 'discord.js' {
     public type: 'text';
     public rateLimitPerUser: number;
     public topic: string | null;
+    private _typing: Map<string, TypingData>;
```

Comment:
bit of a nit, but this could be moved to the TextBasedChannelFields interface, as typing can happen on any TextBasedChannel

Reply:
Ah, I forgot about that, thank you! I've updated the typings so that the property is a part of the `TextBasedChannelFields` interface now, instead of each Channel object individually.